apollo-router/src/plugins/cache/entity.rs

```diff
@@ -34,6 +34,7 @@ use crate::Context;
 
 const ENTITIES: &str = "_entities";
 pub(crate) const REPRESENTATIONS: &str = "representations";
+pub(crate) const CONTEXT_CACHE_KEY: &str = "entity_cache_key";
```

Comment:
Can we namespace it with `apollo.cache::entity_cache_key` for example ?
```suggestion
pub(crate) const CONTEXT_CACHE_KEY: &str = "apollo.cache::entity_cache_key";
```

Reply:
do we use such a convention elsewhere?

Reply:
Ok I'm wrong, if it's an exposed key for the user don't namespace it. But put it in public and also provide it in rhai globals please. I did the same for subscription context key